actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -489,11 +497,26 @@ def []=(name, options)
         def request; @parent_jar.request; end
 
       private
-        def expiry_options(options)
-          if options[:expires].respond_to?(:from_now)
-            { expires_in: options[:expires] }
+        def fetch_metadata(options)
```

Comment:
Why is this named `fetch`? How are we fetching the metadata? And is it the input that's the most important thing here? Do we regularly call methods to pass something in or to get something out?